lib/hanami/logger.rb

```diff
@@ -214,17 +215,33 @@ def _format(hash)
 
       # @api private
       def _format_error(result, hash)
-        result << " #{hash[:error]}:" if hash.key?(:error)
-        result << " #{hash[:message]}#{NEW_LINE}"
+        error_message = if tty? Hanami::Utils::ShellCode.colorize(
+          [ hash[:error], hash[:message] ].compact.join(": "),
+          color: :red,
+          )
+        else
+          [ hash[:error], hash[:message] ].compact.join(": ")
+        end
+        result << [" ", error_message, NEW_LINE].join
         if hash.key?(:backtrace)
           hash[:backtrace].each do |line|
-            result << "from #{line}#{NEW_LINE}"
+            if tty?
+              result << Hanami::Utils::ShellCode.colorize("from #{line}#{NEW_LINE}", color: :yellow)
+            else
+              result << "from #{line}#{NEW_LINE}"
+            end
           end
         end
 
         result
       end
 
+      # @since x.x.x
+      # @api private
+      def tty?
+        @logdev.dev.tty?
```

Comment:
just a question, but why `dev`? will color work for production and test envs too?

Reply:
@jodosha thanks for a link, I wrote this before reading your comment 👍